styles/main.css

```
@@ -0,0 +1,244 @@
+.logo {
+  font-size: 2rem;
+  background-color: transparent;
+  padding: 0;
+}
+
+.movie-search {
+  position: absolute;
+  left: 50%;
+  transform: translateX(-50%);
+
+  display: flex;
```

Stylelint (17.5.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)